backup-loop.sh

```diff
@@ -344,18 +369,52 @@ rclone() {
   }
   backup() {
     ts=$(date +"%Y%m%d-%H%M%S")
-    outFile="${DEST_DIR}/${BACKUP_NAME}-${ts}.${backup_extension}"
+    outFile="${DEST_DIR}/${BACKUP_NAME}-${ts}.tar"
     log INFO "Backing up content in ${SRC_DIR} to ${outFile}"
-    command tar "${excludes[@]}" "${tar_parameters[@]}" -cf "${outFile}" -C "${SRC_DIR}" . || exitCode=$?
+
+    # backup dat and dat_old files separately
+    (cd "${SRC_DIR}" && find . -type f -name "*.dat" -o -name "*.dat_old" ) | command tar "${excludes[@]}" -cf "${outFile}" -C "${SRC_DIR}" -T - || exitCode=$?
     if [ ${exitCode:-0} -eq 1 ]; then
-      log WARN "tar exited with code 1. Ignoring"
+      log ERROR "dat files changed during backup. Retrying in 5 seconds"
+      sleep 5
+      exitCode=0
+
+      (cd "${SRC_DIR}" && find . -type f -name "*.dat" -o -name "*.dat_old" ) | command tar "${excludes[@]}" -cf "${outFile}" -C "${SRC_DIR}" -T - || exitCode=$?
```

Comment:
It seems like this rclone variant is handling the scenario differently with a retry section here.